packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarDate.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import {
+  BpkCalendarDate,
+  BpkCalendarDatePropTypes,
+} from 'bpk-component-calendar';
+
+class BpkScrollableCalendarDate extends PureComponent {
+  render() {
+    const { date, isOutside, ...rest } = this.props;
```

Comment:
No need to pull our `date` explicitly, it'll be passed through `{...rest}`, right?